litmus/director/openebs-install/test.yml

```diff
@@ -0,0 +1,171 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+  
```

Comment:
Put all the tasks into a block.

Reply:
@harshshekhar15 can you please explain how to put all the tasks in a block

Reply:
Added block for tasks @harshshekhar15 
Block is used to group multiple tasks @AmitKumarDas

Reply:
@AmitKumarDas `block and rescue` is used for error handling. If any of the task defined within block fails it goes to the rescue and sets the Flag as failed. It's similar to `try and catch`.